packages/plugin-pnp/sources/index.ts

```diff
@@ -83,6 +85,15 @@ const plugin: Plugin<CoreHooks & StageHooks> = {
       type: SettingsType.STRING,
       default: `pnp`,
     },
+    nodeLinkerFolderLinkMode: {
+      description: `If set to "classic" Yarn will use symlinks on Linux and MacOS and Windows "junctions" on Windows when linking workspaces into "node_modules" directories. This can result in inconsistent behavior on Windows because "junctions" are always absolute paths while "symlinks" may be relative. Set to "symlinks", Yarn will utilize symlinks on all platforms which enables links with relative paths paths on Windows.`,
+      type: SettingsType.STRING,
+      values: [
+        NodeLinkerFolderLinkMode.CLASSIC,
+        NodeLinkerFolderLinkMode.SYMLINKS,
+      ],
+      default: NodeLinkerFolderLinkMode.CLASSIC,
+    },
```

Comment:
- If this is a Windows-only setting, what about calling it `winLinkType: symlinks | junctions`?
- The description in this file should remain short; it's shown in `yarn config -v` and is supposed to be one-line (we already have a couple others that are too long, and it's quite annoying).

Reply:
Sounds good to me. 

@arcanis what about the placement of the option? Should it remain with the pnp plugin as it applies to optional node-linkers that create node_modules folders in pnp mode, or should it go somewhere else?

Reply:
I'd probably move the setting definition directly into the core, since it's no longer tied to linkers in particular 🤔

Reply:
OK, I'll do that. I was a little concerned this might imply that all Yarn functions will respect the setting, and I'm not sure what other (non-linker) functions might use symlinks.

That said, the string 'junction' only appears in the code base in the pnpm and nn linkers so I don't think junctions are used anywhere else...

Reply:
Shortened and added text per @larixer 's suggestion to the docs/Gatsby json